packages/hardhat/package.json

```diff
@@ -29,8 +29,10 @@
     "cids": "1.1.9",
     "dotenv": "10.0.0",
     "envalid": "7.2.1",
+    "hardhat": "2.8.0",
     "ipfs-http-client": "52.0.3",
-    "ipfs-only-hash": "4.0.0"
+    "ipfs-only-hash": "4.0.0",
+    "prettier": "2.5.1"
```

Comment:
Prettier is already installed for the whole project. Look at the main package.json